core/src/main/java/org/apache/carbondata/core/util/path/HDFSLeaseUtils.java

```diff
@@ -164,22 +157,25 @@ private static boolean recoverLeaseOnFile(String filePath, Path path, Distribute
   private static int getLeaseRecoveryRetryCount() {
     String retryMaxAttempts = CarbonProperties.getInstance()
         .getProperty(CarbonCommonConstants.CARBON_LEASE_RECOVERY_RETRY_COUNT,
-            CARBON_LEASE_RECOVERY_RETRY_COUNT_DEFAULT);
+            CarbonCommonConstants.CARBON_LEASE_RECOVERY_RETRY_COUNT_DEFAULT);
     int retryCount = 0;
     try {
       retryCount = Integer.parseInt(retryMaxAttempts);
-      if (retryCount < CARBON_LEASE_RECOVERY_RETRY_COUNT_MIN
-          || retryCount > CARBON_LEASE_RECOVERY_RETRY_COUNT_MAX) {
-        retryCount = Integer.parseInt(CARBON_LEASE_RECOVERY_RETRY_COUNT_DEFAULT);
+      if (retryCount < CarbonCommonConstants.CARBON_LEASE_RECOVERY_RETRY_COUNT_MIN
+          || retryCount > CarbonCommonConstants.CARBON_LEASE_RECOVERY_RETRY_COUNT_MAX) {
+        retryCount = Integer.parseInt(
+            CarbonCommonConstants.CARBON_LEASE_RECOVERY_RETRY_COUNT_DEFAULT);
         LOGGER.warn(
             "value configured for " + CarbonCommonConstants.CARBON_LEASE_RECOVERY_RETRY_COUNT
```

Comment:
change this log to use String.format to avoid string concatenation

Reply:
OK，done